examples/storm-elasticsearch-examples/src/main/java/org/apache/storm/elasticsearch/bolt/EsIndexTopology.java

```diff
@@ -34,52 +39,128 @@
 import org.apache.storm.tuple.Fields;
 import org.apache.storm.tuple.Values;
 
-public class EsIndexTopology {
+/**
+ * Demonstrates an ElasticSearch Strom topology.
+ * @author unknown
```

Comment:
We generally don't use `@author` in the javadoc, it's just noise since most files are worked on by more than one person. Could you remove it here and elsewhere?